lib/cookie.js

```diff
@@ -0,0 +1,81 @@
+/*
+ * derived from jQuery Cookie Plugin v1.4.1
+ * https://github.com/carhartl/jquery-cookies
```

Comment:
This is now https://github.com/carhartl/jquery-cookie

(Looks like he had the same conceptual issue with cookie/cookies.)